pkg/export/export.go

```diff
@@ -40,6 +41,11 @@ import (
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
 )
 
+const (
```

Comment:
Abbreviations like VER probably do more harm than good. I'd also be in favor of explicitly overriding the UA with a single var so these envvars don't get repurposed for a bunch of different purposes over time. Once these start having conflicting requirements it gets messy.